compiler_opt/rl/inlining/inlining_runner.py

```diff
@@ -68,56 +69,52 @@ def compile_fn(
       RuntimeError: if llvm-size produces unexpected output.
     """
 
-    working_dir = tempfile.mkdtemp()
+    working_dir = tempfile.mkdtemp(dir=workdir)
 
     log_path = os.path.join(working_dir, 'log')
     output_native_path = os.path.join(working_dir, 'native')
 
     native_size = 0
-    try:
```

Comment:
why don't we need the try/catch?

Reply:
got it. 

Can you also change the `regalloc_priority_runner.py`?

Reply:
I can do a best effort (since the thing is not in a runnable state right now as far as I can tell).

Reply:
Thanks - mostly about the second inner dir creation, for consistency.